src/runtime_src/core/edge/user/aie/aie.h

```diff
@@ -66,6 +67,15 @@ class Aie {
 
     XAieGbl *getAieInst();
 
+    void
+    sync_bo(uint64_t paddr, const char *dmaID, enum xclBOSyncDirection dir, size_t size);
+
+    void
+    sync_bo_nb(uint64_t paddr, const char *gmioName, enum xclBOSyncDirection dir, size_t size);
+
+    void
+    wait_gmio(const char *gmioName);
```

Comment:
Maybe use std::string in place of const char* for all C++ functions.

Reply:
done